docs/design/latency_definitions/index.md

```diff
@@ -0,0 +1,141 @@
+# Latency definition
+
+CARET mainly measures the following
+
+- Callback latency
+- Communication latency
+- Node latency
+- Path latency
+
+The simplified sequence diagram shown below illustrates each definition.
+
+![Latency overview](../../imgs/latency_overview.png)
+
+Here, the horizontal axis represents time and the vertical axis represents layers.
+The red line represents message flow.
+A message is received in the subscription callback and the processed data is published to the next node.
+In this way, information is propagated from the sensor node to the actuator node.
+
+CARET measures the following three main locations for latency calculation.
+
+- Callback start
+- Callback end
+- Publish
+
+Latency is calculated by calculating the time difference required for each measurement section.
+
+For a more detailed definition, see
+
+- [Callback](./callback.md)
+- [Communication](./communication.md)
+- [Node](./node.md)
+- [Path](./path.md)
+
+CARET can also check other time series data on the Publish side as well.
+All the objects for which time series data can be obtained are listed below.
```

Comment:
```suggestion
All of the objects for which CARET retrieve latency are listed below
```

Reply:
addressed in: [b104708](https://github.com/tier4/CARET_doc/pull/106/commits/b104708b0378c76f80d2ba457892ff9ad0dbfdc4)